docs/node/RPC Category/16-loan.md

```diff
@@ -19,7 +19,10 @@ Creates a loan scheme transaction.
 
 ```ts title="client.loan.createLoanScheme()"
 interface loan {
-  createLoanScheme (scheme: CreateLoanScheme, utxos: UTXO[] = []): Promise<string>
+  createLoanScheme(
```

Comment:
we're using `standard-with-typescript` lint here
https://github.com/DeFiCh/jellyfish/blob/db49cd0d8a38c39f01c83a9936136f8952b344a6/.eslintrc#L6

we need a space in between name and args

![autoformat-ts-standard](https://user-images.githubusercontent.com/8097901/135963189-0ca399b8-925f-42a2-96f7-45b248079529.gif)

Reply:
Oops, seems like my lint rule not applied correctly on md file.